src/workflows-app/RunDetails.js

```diff
@@ -151,6 +156,37 @@ export const BaseRunDetails = (
     [signal, workspaceId]
   );
 
+  const renderInProgressElement = (workflow) => {
+    if (workflow.status === 'Running') {
+      return span({ style: { fontStyle: 'italic' } }, ['In progress - ']);
+    }
+  };
+
+  const renderTaskCostElement = (cost) => {
```

Comment:
Can we apply some DRY and move this function to a place the call table can also use it?